### PR TITLE
Share your story box appears on guide page & story page

### LINKED ISSUE #336: Share your story box appears on guide page & story page
## User story

We should be having this call to action but need to think of a relatable way to attach it to guides. And it will have text related to the guide like "Did you have success in opening a bank account? Tell us how you did it!" 

The story one is slightly easier because we already have "Who":
e.g. "A big thank you to [WHO} for sharing this story. If you have a story like this to share, we would love to hear from you."

This is essential as a way to encourage people to share related stories so that others can gain confidence etc form them.

Example idea for a Guide page:
image

Example idea for a Story page:
image

## Acceptance Criteria

- [x] In the CMS, there is a "template" for adding "Did you achieve the thing in this guide?... Share..." so that there is less of a translation burden.
- [x] The first sentence (in the image) comes from the optional CMS field. The rest is always there & the same for every story or guide.
- [x] On a single guide page, I can see a bespoke / related Call to action to submit my story.
- [x] On a single story page, I can see a bespoke / related Call to action to submit my story.

### PR DESCRIPTION
**Pull request by aaaaargZombies**
Fixes #336 

## Description

- new field added to guide/story records
- new widget added to the cms for each type
- falls back to homepage content if none supplied

these are the two entries with modified content for testing

guide - https://336-share-your-story-box.team-wilder-proto.pages.dev/guides/test-guide-with-all-content
story - https://336-share-your-story-box.team-wilder-proto.pages.dev/stories/test-story-with-minimal-fields

@geeksforsocialchange/developers

### CODE REVIEW

File: public/admin/config.yml [hunk 1]
```diff
@@ -75,6 +75,13 @@ collections:
           pattern: ["^.{50,160}$", "Must be between 50 and 160 characters"],
           i18n: true,
         }
+      - {
+          label: "Call for story 1st sentance",
```
Comment (katjam) [inline public/admin/config.yml:79]:
```suggestion
          label: "Call for story 1st sentence",
```

File: public/admin/config.yml [hunk 2]
```diff
@@ -183,6 +190,13 @@ collections:
         display_fields: ["title"]
         required: false
         i18n: false
+      - {
+          label: "Call for story 1st sentance",
```
Comment (katjam) [inline public/admin/config.yml:194]:
```suggestion
          label: "Call for story 1st sentence",
```

File: src/Page/Guide/View.elm [hunk 3]
```diff
@@ -35,7 +39,7 @@ view language guide allGuides allStories =
                   , viewPrintGuide language
                   , viewRelatedGuideTeasers language guide.relatedGuideList allGuides
                   ]
-                , [ viewRelatedStoryTeasers language guide.relatedStoryList allStories ]
+                , [ viewRelatedStoryTeasers language guide.relatedStoryList allStories, Page.Shared.View.viewCallForStory language call ]
```
Comment (katjam) [inline src/Page/Guide/View.elm:42]: might be easier to read on 2 lines

**Review by katjam — CHANGES_REQUESTED** [review]
Couple typos but looks great

File: src/Page/Guide/View.elm (hunk 3 above)
Comment (aaaaargZombies) [reply src/Page/Guide/View.elm]: I've changed it, I've generally been trusting the formatter to make these decisions.

**Review by ivan-kocienski-gfsc — APPROVED** [review]
Looks good to me

**Review by katjam — APPROVED** [review]

**Pull request merged**